src/types/dataTypes/drupal/node.ts

```diff
@@ -267,6 +270,136 @@ export interface NodePersonProfile extends DrupalNode {
   field_suffix: string
 }
 
+/** A individual event published by a facility.
+ *
+ *  @see https://prod.cms.va.gov/admin/structure/types/manage/event/fields
+ */
+export interface NodeEvent extends DrupalNode {
+  field_additional_information_abo: {
+    value: string
+    format: string
+    processed: string
+  }
+  field_address?: {
+    additionalName?: string | null
+    addressLine1?: string
+    addressLine2?: string
+    administrativeArea?: string
+    countryCode?: string
+    dependentLocality?: string | null
+    familyName?: string | null
+    givenName?: string | null
+    langcode?: string
+    locality?: string
+    organization?: string | null
+    postalCode?: string | null
+    sortingCode?: string | null
+  }
+  field_location_humanreadable: string
+  field_event_cta: string
+  field_event_cost: string
+  field_datetime_range_timezone: DateTimeRange[]
+
+  field_facility_location: FacilityLocation
+  field_featured: boolean
+  field_body: {
+    value: string
+    format: string
+    processed: string
+  }
+  field_include_registration_info: boolean
+  field_location_type: string
+  field_order: string
+  field_publish_to_outreach_cal: boolean
+  field_event_registrationrequired: boolean
+
+  field_description: string
+  field_link: fieldLink
+  field_url_of_an_online_event: urlOfOnlineEvent
+  field_listing: fieldEventListing
+}
+
+interface fieldLink {
+  url: {
+    path: string
+  }
+}
+
+export interface DateTimeRange {
+  value: string
+  end_value: string
+  duration: number
+  rrule: number
+  rrule_index: number
+  timezone: string
+}
+
+export interface urlOfOnlineEvent {
+  uri: string
+  title: string
+  options: unknown
+}
+
+export interface fieldEventListing {
```

Comment:
this type should probably be a NodeEventListing, similar to how NodeNewsStory & NodeStoryListing. If it's extending DrupalNode, a lot of those values won't need to be repeated here (`type`, `id`, etc)